integration/docker/docker_arch_base.go

```diff
@@ -0,0 +1,6 @@
+//+build !s390x
```

Comment:
I don't think we generally use in-file golang build constraints in kata - the constraint via the filename _extention is our normal way. wdty @jodh-intel ?

Reply:
Yes, but you can't negate using the filename afaik, so this one is fine.